service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/testingsupport/email/SendEmailResponseAssert.java

```diff
@@ -20,8 +20,8 @@ public SendEmailResponseAssert hasSubject(String expectedSubject) {
 
     public SendEmailResponseAssert hasBody(EmailContent emailContent) {
         isNotNull();
-        String actualBody = actual.getBody();
-        String expectedBody = emailContent.body();
+        String actualBody = actual.getBody().replaceAll("\\P{Print}", "");
+        String expectedBody = emailContent.body().replaceAll("\\P{Print}", "");
```

Comment:
Had an annoying issue where the assertion failed despite the text appearing identical, I suspect this might be due to some hidden character. 

This change is meant to strip out the non printable characters so hidden characters don't cause tests to fail.